_posts/2019-11-07-snowplow-javascript-tracker-2.12.0-released/index.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: post
+title: "Snowplow JavaScript Tracker 2.12.0 released"
+title-short: Snowplow JavaScript Tracker 2.12.0
+tags: [snowplow, javascript, micro, tracker]
+author: Paul
+category: Releases
+permalink: /blog/2019/11/07/snowplow-javascript-tracker-2.12.0-released/
+discourse: true
+---
+
+We are pleased to announce a new release of the [Snowplow JavaScript Tracker][js-tracker]. [Version 2.12.0][2.12.0-tag] introduces a brand new test suite powered by Snowplow Micro, serving as an example of how Snowplow Micro can be integrated into a testing environment and to give us even more confidence as we continue to release updates to the Snowplow JavaScript Tracker.
```

Comment:
Minor, but I'd reverse the benefits. P1 is more confidence in releases. P2 is the example of using Micro.